docs/source/entities.mdx

```diff
@@ -205,53 +207,352 @@ type Query {
 
 ## Migrating entities and fields (advanced)
 
-As your federated graph grows, you might decide that you want an entity (or a particular field of an entity) to originate in a different subgraph. Apollo Gateway helps you perform these migrations safely.
+As your federated graph grows, you might decide that you want an entity (or a particular field of an entity) to originate in a different subgraph. This section describes how to perform these migrations.
 
 ### Entity migration
 
-Let's say our `payments` subgraph defines a `Bill` entity. Then, we add a dedicated `billing` subgraph to our federated graph. It now makes sense for the `Bill` entity to originate in the `billing` subgraph instead.
+Let's say our Payments subgraph defines a `Bill` entity:
+
+<CodeColumns>
+
+```graphql
+# Payments subgraph
+type Bill @key(fields: "id") {
+  id: ID!
+  amount: Int!
+}
+
+type Payment {
+  # ...
+}
+```
+
+</CodeColumns>
+
+Then, we add a dedicated Billing subgraph to our federated graph. It now makes sense for the `Bill` entity to originate in the Billing subgraph instead. When we're done migrating, we want our deployed subgraph schemas to look like this:
+
+<CodeColumns>
+
+```graphql
+# Payments subgraph
+type Payment {
+  # ...
+}
+```
+
+```graphql
+# Billing subgraph
+type Bill @key(fields: "id") {
+  id: ID!
+  amount: Int!
+}
+```
+
+</CodeColumns>
+
+The exact steps depend on how you perform schema composition:
+
+<ExpansionPanel title="With Rover CLI composition">
+
+1. In the Billing subgraph's schema, define the `Bill` entity just as it's defined in the Payments subgraph (do _not_ extend it):
+
+    <CodeColumns>
+
+    ```graphql
+    # Payments subgraph
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+
+    type Payment {
+      # ...
+    }
+    ```
+
+    ```graphql{2-5}
+    # Billing subgraph
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+    ```
 
-We can perform this migration safely with the following steps:
+    </CodeColumns>
 
-1. In the `billing` subgraph's schema, define the `Bill` entity just as it's defined in the `payments` subgraph (do _not_ extend it). If you're using managed federation, register this schema change with Studio.
+    * Note that if you perform composition at this point, it produces an error because the `Bill` entity can't originate in more than one subgraph. That's okay! We'll resolve this error in a couple of steps.
```

Comment:
Maybe, for extra clarity, something like: "That's okay, we don't run composition at this point and we'll resolve this error ..."?